src/core/configuration.c

```diff
@@ -47,6 +48,7 @@ static struct argp_option options[] = {
   {"ckpt-period",         CKPT_PERIOD_KEY          , "#EVENTS" ,  0                  ,  "Number of events to be forward-executed before taking a full-snapshot"   , 0 },
   {"ckpt-fossil-period",  CKPT_FOSSIL_PERIOD_KEY   , "#EVENTS" ,  0                  ,  "Number of events to be executed before collection committed snapshot"   , 0 },
   {"ckpt-autonomic-period",  CKPT_AUTONOMIC_PERIOD_KEY, 0         ,  OPTION_ARG_OPTIONAL,  "Enable autonomic checkpointing period"   , 0 },
+  {"ckpt_forced_full_period",  CKPT_FORCED_FULL_PERIOD_KEY, 0         ,  OPTION_ARG_OPTIONAL,  "Number of incremental checkpoints before taking a full log"   , 0 },
```

Comment:
we need a value for ckpt_forced_full_period

```suggestion
  {"ckpt_forced_full_period",  CKPT_FORCED_FULL_PERIOD_KEY, "#EVENTS"         ,  0,  "Number of incremental checkpoints before taking a full log"   , 0 },
```